app/models/concerns/foreman_bootdisk/host_ext.rb

```diff
@@ -33,6 +33,14 @@ def bootdisk?
     managed? && bootdisk_build? && SETTINGS[:unattended]
   end
 
+  def bootdisk_downloadable?
+    architecture.blank? || intel_arch?
+  end
+
+  def intel_arch?
```

Comment:
No need to extend the model in this plugin, it's unnecessary complexity when it's only used by the UI helper.